### ISSUE TITLE
Update the README.md

### ISSUE THREAD

**Issue by claire-me** [Covisource/covisource-web#5]
The README.MD needs steps on how to contribute to  Covisource-web. Steps like tow to clone, push, and make a pull request

**Comment by fullstackslayer** [comment]
Yes. Would you like to work on that? We also need instructions on what to put in environment variables.

**Comment by fullstackslayer** [comment]
@claire-me would you be interested on editing the readme

**Comment by claire-me** [comment]
> Yes. Would you like to work on that? We also need instructions on what to put in environment variables.

I don't understand
